modal/__main__.py

```diff
@@ -65,6 +65,8 @@ def main():
         else:
             title = "Error"
             content = str(exc)
+            if notes := getattr(exc, "__notes__", []):
+                content = f"{content}\n\nNote: {' ' .join(notes)}"
```

Comment:
nice, that's helpful! Minor but does it make sense to emphasize this with Rich markup? Sometimes people's eyes gloss over when there's a lot of exception stuff. Maybe not necessary since this is already in a box.